build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -62,6 +131,91 @@ libev:
+mate-control-center:
+  :rpm_name: mate-control-center-1.16.1-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-control-center-1.16.1-1.el7.x86_64.rpm
+mate-control-center-filesystem:
+  :rpm_name: mate-control-center-filesystem-1.16.1-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-control-center-filesystem-1.16.1-1.el7.x86_64.rpm
+mate-desktop:
+  :rpm_name: mate-desktop-1.16.2-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-desktop-1.16.2-1.el7.x86_64.rpm
+mate-desktop-libs:
+  :rpm_name: mate-desktop-libs-1.16.2-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-desktop-libs-1.16.2-1.el7.x86_64.rpm
+mate-icon-theme:
+  :rpm_name: mate-icon-theme-1.16.2-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-icon-theme-1.16.2-1.el7.noarch.rpm
+mate-menus:
+  :rpm_name: mate-menus-1.16.0-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-menus-1.16.0-1.el7.x86_64.rpm
+mate-menus-libs:
+  :rpm_name: mate-menus-libs-1.16.0-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-menus-libs-1.16.0-1.el7.x86_64.rpm
+mate-notification-daemon:
+  :rpm_name: mate-notification-daemon-1.16.1-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-notification-daemon-1.16.1-1.el7.x86_64.rpm
+mate-panel:
+  :rpm_name: mate-panel-1.16.2-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-panel-1.16.2-1.el7.x86_64.rpm
+mate-panel-libs:
+  :rpm_name: mate-panel-libs-1.16.2-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-panel-libs-1.16.2-1.el7.x86_64.rpm
+mate-polkit:
+  :rpm_name: mate-polkit-1.16.0-2.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-polkit-1.16.0-2.el7.x86_64.rpm
+mate-power-manager:
+  :rpm_name: mate-power-manager-1.16.2-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-power-manager-1.16.2-1.el7.x86_64.rpm
+mate-screensaver:
+  :rpm_name: mate-screensaver-1.16.1-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-screensaver-1.16.1-1.el7.x86_64.rpm
+mate-session-manager:
+  :rpm_name: mate-session-manager-1.16.1-3.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-session-manager-1.16.1-3.el7.x86_64.rpm
+mate-settings-daemon:
+  :rpm_name: mate-settings-daemon-1.16.2-1.el7.x86_64.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-settings-daemon-1.16.2-1.el7.x86_64.rpm
+mate-themes:
+  :rpm_name: mate-themes-3.22.16-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-themes-3.22.16-1.el7.noarch.rpm
+mate-user-guide:
+  :rpm_name: mate-user-guide-1.16.0-1.el7.noarch.rpm
+  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/m/mate-user-guide-1.16.0-1.el7.noarch.rpm
+nx-libs:
```

Comment:
new version: nx-libs-3.5.99.16-1.el7.x86_64.rpm 
                     nxagent, and nxproxy also. (3.5.99.16-1 also.)